src/content/competitions/HiPAC2023.md

```diff
@@ -0,0 +1,34 @@
+
+The competition tested our skills in performance tuning for HPL, HPCG, atmospheric applications (FourCastNet), and fluid dynamics applications (Basilisk). The competition spanned three days, from 2023 August 8th to August 10th, and required us to address various computational challenges, including data preprocessing, cross-node model training, simulation visualization, and performance evaluation.
+
+While we had theoretical knowledge of many HPC concepts, this competition required us to apply that knowledge practically. Installing and configuring Basilisk for parallel computing, optimizing model training for FourCastNet, and ensuring efficient data preprocessing were all tasks that demanded practical solutions. This hands-on experience was invaluable in bridging the gap between theory and practice.
+
+The competition underscored the importance of teamwork and effective communication. With tight deadlines and complex problems to solve, we had to rely on each other's strengths and work collaboratively. Our success was a collective effort, with each team member contributing unique insights and expertise. The experience strengthened our ability to work as a cohesive unit, making quick decisions and adapting to challenges as they arose.
+
+One of the most rewarding aspects of the competition was overcoming the various challenges we encountered. From debugging installation issues to fine-tuning our simulations and models, each obstacle taught us resilience and problem-solving skills. The sense of accomplishment after resolving a particularly tough issue was immensely gratifying and fueled our drive to keep pushing forward.
+
+This competition highlighted the critical role of HPC in solving real-world problems. Whether it was simulating complex fluid dynamics or predicting weather patterns using machine learning, HPC proved to be an indispensable tool. It reinforced our belief in the potential of HPC to drive innovation and solve pressing scientific and engineering challenges.
+
+We are deeply grateful for the support provided by the National Center for High-performance Computing. Access to their Taiwania2 and Taiwania3 clusters was crucial for testing and refining our solutions. 
+
+Participating in the HPC competition was a challenging yet rewarding experience that enriched our technical skills, fostered teamwork, and deepened our appreciation for the power of high-performance computing. We are proud of our achievements and look forward to leveraging this experience in our future endeavors. This competition was not just about winning but about learning, growing, and pushing the limits of what we can achieve with technology.
+
+Competition Winners Announcement: [link](https://event.nchc.org.tw/2023/hipac/awards.html)
+
+- Team-2 Members: SHIH-JOU CHENG, WEI-CHIH HUANG, SSU-CHENG Lai, WEI-PO LIN, TUNG-YU HSIEH, CHIH-YU HSIEH
```

Comment:
maybe
"SSU-CHENG Lai", should be replaced with "SSU-CHENG LAI"

Reply:
fixed it!